### DISCUSSION TITLE
Scanning one day of CCTV produced 765gb or 900,000 files

### DISCUSSION THREAD

**Discussion by 9812ga98sd** [Breakthrough/DVR-Scan#130] [Q&A]
Scanning one day of CCTV produced 765gb or 900,000 files
dvr-scan -i input/*.mp4 -d output/ -b MOG2_CUDA

Is the sensitivity too high? I left it at default.
Windows 10

**Comment by Breakthrough** [comment]
It depends on your use case.  The defaults aren't really tuned on a large dataset, so tweaking the sensitivity may be required in your case.  You may also want to limit the detection region.

See the documentation for the [various detector options](https://dvr-scan.readthedocs.io/en/latest/guide/options/#detection-parameters) for a starting point for tuning for your application.  Hope this helps, thanks.
